contrib/completion/bash/docker

```diff
@@ -4925,13 +4871,11 @@ _docker_stack_deploy() {
 
 	case "$cur" in
 		-*)
-			local options="--compose-file -c --help --orchestrator"
-			__docker_stack_orchestrator_is kubernetes && options+=" --kubeconfig --namespace"
-			__docker_stack_orchestrator_is swarm && options+=" --prune --resolve-image --with-registry-auth"
+			local options="--compose-file -c --help --prune --resolve-image --with-registry-auth"
```

Comment:
Same here, the $option variables should only exist if the option list is constructed incrementally.